src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -353,7 +436,15 @@ function WebGLShadowMap( _renderer, _objects, maxTextureSize ) {
 		result.visible = material.visible;
 		result.wireframe = material.wireframe;
 
-		result.side = ( material.shadowSide != null ) ? material.shadowSide : shadowSide[ material.side ];
+		if ( type === VSMSoftShadowMap ) {
+
+			result.side = ( material.shadowSide != null ) ? material.shadowSide : material.side;
```

Comment:
I will trust @Mugen87's  review of the implementation, but can you please explain this change?

Reply:
Sure, normal shadowmaps benefit from back side rendering to prevent self occlusion but that's not an issue with VSM, but it does depend on correct occlusion depth so backside rendering results in peter panning and other strange artefacts in VSM. Hence the conditional statement to distinguish.